packages/react/menu/src/Menu.tsx

```diff
@@ -572,10 +571,12 @@ const MenuContentImpl = React.forwardRef((props, forwardedRef) => {
                       searchRef.current = '';
                     }
                   })}
-                  onMouseMove={composeEventHandlers(props.onMouseMove, (event) => {
-                    const target = event.target as HTMLElement;
-                    if (event.currentTarget.contains(target) && event.movementX !== 0) {
-                      pointerDirRef.current = event.movementX > 0 ? 'right' : 'left';
+                  onPointerMove={composeEventHandlers(props.onPointerMove, (event) => {
+                    if (event.pointerType === 'mouse') {
+                      const target = event.target as HTMLElement;
+                      if (event.currentTarget.contains(target) && event.movementX !== 0) {
+                        pointerDirRef.current = event.movementX > 0 ? 'right' : 'left';
+                      }
```

Comment:
Do you think it would be cleaner to wrap these conditions up in a self documenting util? [we did something like this before](https://github.com/radix-ui/primitives/blob/ceeb027884a110611467d91287baf47cf6fccfdf/packages/react/hover-card/src/HoverCard.tsx#L196-L199)

What do you think?

Reply:
I like that. There are 3 pointer inputs right? `mouse`, `pen`, `touch`. How should we treat `pen`? My guess would be that `pen` is basically a less coarse version of `touch` so we should treat them the same?

Reply:
Yeah I'd agree with that, at least in our general cases.